core/service/src/engine/threshold/service/public_decryptor.rs

```diff
@@ -333,17 +333,16 @@ impl<
                     "Public decryption request with ID {req_id} already exists"
                 )));
             }
-
-            if !ok_or_tonic_abort(
-                self.crypto_storage.threshold_fhe_keys_exists(&key_id).await,
```

Comment:
I think the method `threshold_fhe_keys_exists` can be removed now, as it no longer seems to be in use

Reply:
There are several other similar methods (for CRS and other keys) that were added recently [here](https://github.com/zama-ai/kms/commit/58332d39d12346f7b83b790dca9a3e0cd5df6bc3) and [here](https://github.com/zama-ai/kms/commit/e70bf106cf6c4ef0b99f632297caa1c8f8084373).

I propose to keep this PR as simple fix and then do a proper and more detailed analysis and cleanup of the functions in a followup, when Maksym and Kelong are back.

Reply:
Sure, sound good to me! But can I get you to make a follow-up issue to make sure we don't forget the changes needed :)